src/deprecated/platformSpecificDeprecated.ios.js

```diff
@@ -88,6 +88,7 @@ function startTabBasedApp(params) {
           style={params.tabsStyle}
           appStyle={params.appStyle}
           initialTabIndex={params.initialTabIndex}>
+          overlay={params.overlay}
```

Comment:
typo here, the React Native element is already closed before the extra prop is included. This causes a crash whenever a tab app is started.